### ISSUE TITLE
ShotMem or Macro trigger/recall on AV-UHS500

### ISSUE THREAD

**Issue by WhatTheNasus** [bitfocus/companion-module-panasonic-avhs#9]
Is it possible to also add the functionality to recall from shot memory or macro memory?

I've seen in the manual that the "Macro trigger" is possible but I'm not sure about the other one.

**Comment by T5R4C3R** [comment]
I second this request. Shot Memory Recall would be a game changer for this switcher!! I don’t see anything in the Panasonic External Protocol manual though. I don’t think it’s possible.

**Comment by ericbusby77** [comment]
I use this module with the AV-HS410, and while I cannot recall Shot Memories, I can pretty much recreate them by just programming the parts of the shot memory I want into the button in Companion.

**Comment by T5R4C3R** [comment]
I don’t think it’s possible to trigger shot memories. One of Panasonics Sales engineers just told me so. He says Shot memories are multiple commands combined into one and are triggered internally by the switcher. lol. Of course they are, that’s why we want to trigger them! 

I think I’m going to have to use Companion for what it can do, and also use Shot memories for my PIP animated looks that I use.

**Comment by T5R4C3R** [comment]
I found this in the back of the UHS500 Manual. Are we able to trigger the SHOTPLAY with these ID numbers? I am away from my switcher and cant test at the moment <img width="1058" alt="Screenshot 2024-09-10 at 7 58 50 AM" src="https://github.com/user-attachments/assets/dc367714-927d-4830-a320-48a3da738d24">

**Comment by tdg5591** [comment]
So to clarify, macro recall is not currently a pre-defined feature in companion, I'm not just missing it?  This feature would be awesome if it was in companion!!  How can we get it?

**Comment by T5R4C3R** [comment]
Exactly. It is available for Lawo 3rd party external control. That is where all of these extra commands come into play. I am waiting for LAWO to get back to me with a price. I am sure it is going to be higher than I am willing to pay.
